config/set/contentrepository-90.php

```diff
@@ -379,9 +380,43 @@
     // Context::getNodesOnPath()
     // TODO: PHP
     // Context::adoptNode()
+    // TODO: PHP
     // Context::getFirstLevelNodeCache()
     $rectorConfig->rule(ContextGetFirstLevelNodeCacheRector::class);
+    // getCurrentDateTime(): DateTime|DateTimeInterface
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.currentDateTime', 'Line %LINE: !! node.context.currentDateTime is removed in Neos 9.0.');
+    // getDimensions(): array
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.dimensions', 'Line %LINE: !! node.context.dimensions is removed in Neos 9.0. You can get node DimensionSpacePoints via node.dimensionSpacePoints now.');
+    // getNodeByIdentifier(identifier: string): NodeInterface|null
+    // TODO: PHP
+    // getProperties(): array
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.properties', 'Line %LINE: !! node.context.properties is removed in Neos 9.0.');
+    // getTargetDimensions(): array
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.targetDimensions', 'Line %LINE: !! node.context.targetDimensions is removed in Neos 9.0.');
+    // getTargetDimensionValues(): array
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.targetDimensionValues', 'Line %LINE: !! node.context.targetDimensionValues is removed in Neos 9.0.');
+    // getWorkspace([createWorkspaceIfNecessary: bool = true]): Workspace
+    // TODO: PHP
+    // TODO: Fusion?
```

Comment:
as discussed here id say write a comment to create a custom eel helper to replace `context.workspace.*` https://github.com/neos/rector/issues/44#issuecomment-2573058752

Reply:
I've added the rector to rewrite them.